src/storage/mod.rs

```diff
@@ -18,6 +22,7 @@ use bincode::rustc_serialize::{ EncodingError,DecodingError,encode_into,decode_f
 /// constants
  const MAGIC_NUMBER: u64 = 0x6561742073686974; // secret
  const MAGIC_NUMBER_BYTES: usize = 8;
```

Comment:
You can obtain the size of a type in bytes with `std::mem::size_of_val`. So there is no need to use this constant.